push_notifications/admin.py

```diff
@@ -2,18 +2,17 @@
 from django.contrib import admin, messages
 from django.utils.translation import ugettext_lazy as _
 from .gcm import GCMError
-from .models import APNSDevice, GCMDevice, get_expired_tokens
+from .models import APNSDevice, GCMDevice, WNSDevice, get_expired_tokens
 from .settings import PUSH_NOTIFICATIONS_SETTINGS as SETTINGS
 
-
 User = apps.get_model(*SETTINGS["USER_MODEL"].split("."))
 
 
 class DeviceAdmin(admin.ModelAdmin):
 	list_display = ("__str__", "device_id", "user", "active", "date_created")
-	list_filter = ("active", )
+	list_filter = ("active",)
 	actions = ("send_message", "send_bulk_message", "prune_devices", "enable", "disable")
-	raw_id_fields = ("user", )
+	raw_id_fields = ("user",)
```

Comment:
You're making unnecessary whitespace changes all over the place - these pollute the diffs and make it very hard to review the PR. Please don't do that.